src/test/java/edu/hw1/Task8Test.java

```diff
@@ -52,7 +52,7 @@ void knightBoardCapture_3() {
         assertFalse(response);
     }
     @Test
-    void knightBoardCapture_4() {
+    void knightBoardCaptureFiguresNotContact() {
```

Comment:
`BUG` в джаве название метода теста называют в snake_case, или полу snake_case для читаемости Например
```suggestion
    void knightBoard_CaptureFigures_NotContact() {
```